src/Build.OM.UnitTests/Definition/Project_Tests.cs

```diff
@@ -4362,5 +4362,10 @@ private void VerifyContentOfSampleProject(Project project)
             Assert.Equal("$(p)X;i3", items[1].UnevaluatedInclude);
             Assert.Equal("i3", items[2].EvaluatedInclude);
         }
+
+        private bool FileSystemIsCaseSensitive()
+        {
+            return !(RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
```

Comment:
Isn't the second half of this redundant?

Also, I don't like having these implementations. Could you just call the version in FileUtilities? And then have the version in FileUtilities call the BCL version instead of copying their code? That second half may need help with making internals visible if it's possible.

Reply:
My understanding is that the BCL implementation isn't publicly available, but Nick is following up on that in the other thread. 
Otherwise, I'll use the version from FileUtilities. It feels a little uncomfortable since tests won't catch problems with the FileUtilities implementation but I suppose BCL's implementation is safe.